public/data/aroramrinaal.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Mrinaal Arora",
+  "type": "personal",
+  "bio": "OpenSource Contributer",
```

I think Contributor is with `o`

```suggestion
  "bio": "Open Source Contributor",
```

---

Fixed It.